src/main/java/com/github/admissionCommittee/util/init/FacultyInitializerUtil.java

```diff
@@ -14,41 +18,65 @@
 import java.util.Random;
 import java.util.Set;
 
-public class FacultyInitializerUtil implements EntityInitializerUtil<Faculty> {
+import static com.github.admissionCommittee.model.enums.UserTypeEnum.ADMIN;
+
+public class FacultyInitializerUtil implements InitializerUtil {
+    private static final Logger log = LoggerFactory.getLogger
+            (FacultyInitializerUtil.class);
+
     @Override
-    public List<Faculty> initEntities(int entitiesNumber, String inputFile,
-                                      String outputFile) {
+    public void init(int entitiesNumber, String inputFile,
+                     String outputFile) {
         try {
+            FacultyValidatorUtil validator = new
+                    FacultyValidatorUtil();
             List<Faculty> facultyList = new ArrayList<>();
+            final int[] counter = {0};
             Files.lines(Paths.get(inputFile))
                     .map(line -> line.split("\u200B"))
                     .forEach(substring -> {
                         Faculty faculty = new Faculty(substring[2], Integer
                                 .parseInt(substring[3]), getRandomSubject(
                                 3));
-                        ValidatorUtil.getValidator(Faculty.class)
-                                .validateEntity(faculty);
+                        validator.validate(faculty);

```

Comment:
unchecked exception without processing

Reply:
Validation implemented, during init logging to file; all services save method's returns logs of errors; validation of process of user's profile creation based on it (checking the password, name, etc, if failes, user see it in the message).